analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectWithCustomGroovyScriptFunctionalTest.java

```diff
@@ -93,13 +93,19 @@ public void verifyGroovyInjection() throws IOException, URISyntaxException, Mani
         assertTrue(systemOutRule.getLog().contains("Attempting to read URL"));
         assertTrue(systemOutRule.getLog().contains("found new version is 1.0.1.redhat-00002"));
         assertTrue(systemOutRule.getLog().contains("original version is 1.0.1" + System.lineSeparator()));
-
+        assertThat(FileUtils.readFileToString(new File(projectRoot, "build.gradle"), Charset.defaultCharset()))
+                .satisfies(s -> {
+                    assertTrue(s.contains("classpath \"org.hibernate:hibernate-core:5.3.7.Final-redhat-00001"));
+                    assertTrue(s.contains("publishing {\n" +
+                            "  publications {\n" +
+                            "    nebula {\n" +
+                            "      artifact(\"build/distributions/${project.name}-${version}.zip\")\n"));
+                });
```

Comment:
This and the next, it seems you could simplify the logic (and get better error messages) with

```
assertThat(s).contains("foo").contains("bar").doesNotContain("baz");
```

Reply:
I'd be happy with the damn tests passing!